romana-install/roles/provision/create-aws-hosts/templates/aws_cf_small_cluster.json

```diff
@@ -1052,27 +1052,19 @@ under the License.
   "Parameters": {
     "ControllerInstanceType": {
       "Description": "EC2 instance type for Controller Node",
-      "Type": "String",
-      "Default": "t2.large",
-      "ConstraintDescription": "Must be a valid EC2 instance type."
```

Comment:
Not really, but this fairly invisible to a user.
It could be detailed in the point where the value is defined (`group_vars/aws`) rather than spots where it eventually gets used.
If users are playing with the values there, I'd expect them to be sufficiently familiar with EC2 instance names and not really need an explanation.